ONSdigital/sdx-console
# Update sdc rabbit

## What? and Why?
Needed to update sdc-rabbit to fix reconnection issues.

## How to test
- Spin up all the services in sdx-compose
- Test the features of console to make sure they still work (survey submissions and seft both use the queue so they are good to try)
- Restart the rabbit container then try and send a submission after it comes back up

## Checklist
